lib/prompt.js

```diff
@@ -229,10 +229,6 @@ class Prompt extends Events {
     return new Promise(async(resolve, reject) => {
       this.once('submit', resolve);
       this.once('cancel', reject);
-      if (await this.skip()) {
```

Comment:
This code can't be removed. Prompts are designed to work without the `Enquirer` class. However, we can do something like:

```js
if (this.skipped === true || await this.skip()) {
  this.render = () => {};
  return this.submit();
}
```

Reply:
Do we need to add another test case for this or does my added test test this?

Reply:
Also, would we need to emit a `skipped` event here too?